.env.example

```
@@ -8,7 +8,7 @@ NEXT_PUBLIC_IS_MAINNET=false
 NEXT_PUBLIC_RELAY_ON=false
 NEXT_PUBLIC_SENTRY_DSN=""
 NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID=""
-NEXT_PUBLIC_ALCHEMY_KEY=""
+NEXT_PUBLIC_ALCHEMY_KEY="cr42w2-6bUdSw21LfukV8BsQrXZqLCqb"
```

```suggestion
NEXT_PUBLIC_ALCHEMY_KEY="cr42w2-6bUdSw21LfukV8BsQrXZqLCqb"
```
You should probably remove your alchemy key from the .env.example